packages/grafana-ui/src/components/DataLinks/DataLinksContextMenu.tsx

```diff
@@ -14,14 +14,19 @@ export const DataLinksContextMenu: React.FC<DataLinksContextMenuProps> = ({ chil
     return children({});
   }
 
-  const getDataLinksContextMenuItems = () => {
-    return [{ items: linkModelToContextMenuItems(links), label: 'Data links' }];
-  };
+  const items = useMemo(() => linkModelToContextMenuItems(links), [links]);
```

Comment:
Probably no need to memoize this because the `links` function is prob mutated in the DataFrame but added it anyway. Also, if there's another way to figure out if the number of `links` without invoking the function `links` I'd love to hear about how to solve that.

Reply:
yea, think it would be better to have a fast way to know if there are links.